docs/ddl-of-carbondata.md

```diff
@@ -157,22 +157,22 @@ CarbonData DDL statements are documented here,which includes:
      * GLOBAL_SORT: It increases the query performance, especially high concurrent point query.
        And if you care about loading resources isolation strictly, because the system uses the spark GroupBy to sort data, the resource can be controlled by spark. 
 
-    ### Example:
```

Comment:
There are many Examples in the file， we should unify it.

Reply:
> There are many Examples in the file， we should unify it.

ok